src/bot/texts/EN/messages.json

```diff
@@ -18,24 +18,24 @@
   "application_not_found": "❓ Unfortunately, we couldn't find your application. Please ensure you've entered the correct application number and try again. If you've recently submitted the application, please wait for some time until the Ministry of Interior publishes the data about it.\n\nCurrent status {status_sign}:",
   "application_in_progress": "⏳ Your application is still under review.\n\nCurrent status {status_sign}:",
   "application_approved": "🎉 Congratulations! Your application has been approved!\n\nCurrent status {status_sign}:",
-  "application_denied": "😔 Unfortunately, your application was declined ...\n\nCurrent status {status_sign}:",
-  "application_failed": "😥Unfortunately, we couldn't get the status of <b>{app_string}</b> application. Please ensure your application details are correct and try unsubscribing and then subscribing again. If the issue persists, you can reach out to developers.",
-  "current_status": "Current application status is {status_sign}:",
-  "current_status_timestamp": "Current Status {status_sign}:\n\n{status}\n\nLast Updated: <b>{timestamp}</b>",
+  "application_denied": "😔 I regret to tell, your application was declined ...\n\nCurrent status {status_sign}:",
```

Comment:
Unfortunately sounds like a better fit imho.

Reply:
`"😔 I regret to inform that your application was declined. \n\nCurrent status {status_sign}:",`